setup.py

```diff
@@ -178,9 +178,7 @@ def check_for_openmp():
         "url": 'https://khmer.readthedocs.org/',
         "packages": ['khmer', 'khmer.tests', 'oxli'],
         "package_dir": {'khmer.tests': 'tests'},
-        "install_requires": ['screed >= 0.9'],
-        # testing screed download link
-
+        "install_requires": ['screed >= 0.9', 'bz2file'],
```

Comment:
what happens if bz2file isn't available? I'm -0 on making this a hard requirement (although it may come with all the Pythons now).